crawler-cli/src/main/java/org/gbif/crawler/pipelines/dwca/DwCAToAvroCommandVerification.java

```diff
@@ -0,0 +1,144 @@
+package org.gbif.crawler.pipelines.dwca;
+
+import org.gbif.common.messaging.api.messages.DwcaValidationFinishedMessage;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.eclipse.jdt.internal.core.Assert;
```

Comment:
this class must not be used, is not used anywhere in the crawler project and looks to me that it is a eclipse jdt class

Reply:
removed.